src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/grouped/UnpaidAttendanceSummaryReportITest.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.juror.api.moj.report.grouped;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupByResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupedReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.GroupedTableData;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractGroupedReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.DataType;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Sql({
+    "/db/truncate.sql",
+    "/db/mod/truncate.sql",
+    "/db/administration/createUsers.sql",
+    "/db/mod/reports/UnpaidAttendanceSummaryReportITest_typical.sql"
+})
+@SuppressWarnings("PMD.LawOfDemeter")
+class UnpaidAttendanceSummaryReportITest extends AbstractGroupedReportControllerITest {
```

Comment:
Need test for when you are a bureau user (should be rejected)

Reply:
Added